src/main/java/cms/gov/madie/measure/services/MeasureService.java

```diff
@@ -862,6 +862,18 @@ public List<Measure> getQiCoreMeasuresByCmsId(Integer qdmCmsId) {
   }
 
   public void validateCmsIdAssociation(String username, Measure qiCoreMeasure, Measure qdmMeasure) {
+    if (!qiCoreMeasure.getModel().equals(ModelType.QI_CORE.getValue())
+        && !qiCoreMeasure.getModel().equals(ModelType.QI_CORE_6_0_0.getValue())) {
```

Comment:
I think this can be combined with the qdm check with an OR operator. Then we won't have duplicate exception throws and log statements.
```suggestion
    if ((!qiCoreMeasure.getModel().equals(ModelType.QI_CORE.getValue())
            && !qiCoreMeasure.getModel().equals(ModelType.QI_CORE_6_0_0.getValue()))
        || !qdmMeasure.getModel().equals(ModelType.QDM_5_6.getValue())) {
```